js/application/init.js

```diff
@@ -194,7 +194,7 @@ define([
           document.title = this.config.title;
 
           // Launch the app specifics
-          this.main.runMap(this.config, webmap, view);
+          this.main.start(this.config, dom.byId("viewDiv"), view, webmap, null, null);
```

Comment:
@MikeTschudi can we make one object passed as an argument instead of multiple arguments with null values?